frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/NextBranchCheckoutAction.java

```diff
@@ -0,0 +1,27 @@
+package com.virtuslab.gitmachete.frontend.actions.traverse;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import io.vavr.control.Option;
+import lombok.experimental.ExtensionMethod;
+import lombok.val;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.frontend.actions.base.BaseCheckoutAction;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeySelectedBranchName;
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle;
+
+@ExtensionMethod(GitMacheteBundle.class)
+public class NextBranchCheckoutAction extends BaseCheckoutAction
```

Comment:
For consistency:
```suggestion
public class CheckoutNextAction extends BaseCheckoutAction
```
We skip `Branch` from action names, https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/CONTRIBUTING.md#coding-conventions -> last bullet